### DISCUSSION TITLE
Project Boards, feature: Attachment previews / Cover images.

### DISCUSSION THREAD

**Discussion by gnat** [community/community#42015] [Projects and Issues]
### Select Topic Area

Product Feedback, Feature Request

### Body

For issue cards on github projects.

Like Trello. Just submitted a patch to gitea implementing this for gitea projects: https://github.com/go-gitea/gitea/pull/22112

**Comment by PoetaKodu** [comment]
I came here to ask for the same feature. Right now issue list is just a big wall of text.

**Comment by li-phone** [comment]
你好，你的邮件已收到，稍后将会查看，尽快给你回复。

**Comment by gnat** [comment]
@PoetaKodu it's ready to go, just waiting on a merge. Likely in 1.19 https://github.com/go-gitea/gitea/pull/22112

Feel free to post in that PR to help move things along.
